hummingbot/connector/derivative/bybit_perpetual/bybit_perpetual_derivative.py

```diff
@@ -0,0 +1,306 @@
+                                    f"Error: {e} ",
+                                    exc_info=True)
+
+    async def _update_positions(self):
+        """
+        Retrieves all positions using the REST API.
+        """
+        symbol_trading_pair_map: Dict[str, str] = await OrderBookDataSource.trading_pair_symbol_map(self._domain)
+
+        raw_response = await self._api_request(method="GET",
+                                               path_url=CONSTANTS.GET_POSITIONS_PATH_URL,
+                                               is_auth_required=True)
+
+        result: List[Dict[str, Any]] = raw_response["result"]
+
+        for position in result:
+            if not position["is_valid"]:
+                self.logger().error(f"Received an invalid position entry. Position: {position}")
+                continue
+            ex_trading_pair = position.get("symbol")
+            hb_trading_pair = symbol_trading_pair_map.get(ex_trading_pair)
+            position_side = PositionSide.LONG if position.get("side") == "buy" else PositionSide.SHORT
+            unrealized_pnl = Decimal(str(position.get("unrealised_pnl")))
+            entry_price = Decimal(str(position.get("entry_price")))
+            amount = Decimal(str(position.get("size")))
+            leverage = Decimal(str(position.get("effective_leverage")))
+            pos_key = self.position_key(hb_trading_pair, position_side)
+            if amount != s_decimal_0:
+                self._account_positions[pos_key] = Position(
+                    trading_pair=hb_trading_pair,
+                    position_side=position_side,
+                    unrealized_pnl=unrealized_pnl,
+                    entry_price=entry_price,
+                    amount=amount,
+                    leverage=leverage,
+                )
+            else:
+                if pos_key in self._account_positions:
+                    del self._account_positions[pos_key]
+
+    async def _status_polling_loop(self):
+        while True:
+            try:
+                await self._status_poll_notifier.wait()
+                await safe_gather(
+                    # self._update_balances(),
+                    self._update_positions()
+                )
+            except asyncio.CancelledError:
+                raise
+            except Exception as e:
+                self.logger().network(f"Unexpected error while fetching account updates. Error: {e}",
+                                      exc_info=True,
+                                      app_warning_msg=f"Could not fetch account updates from {CONSTANTS.EXCHANGE_NAME} Perpetuals. "
+                                      )
+                await asyncio.sleep(0.5)
+            finally:
+                self._status_poll_notifier = asyncio.Event()
+
+    async def _set_leverage(self, trading_pair: str, leverage: int = 1):
+        ex_trading_pair = bybit_utils.convert_to_exchange_trading_pair(trading_pair)
```

Comment:
We should not use the `convert_to_exchange_trading_pair` logic in the utils module, since the order book data stream has the pairs mappings